### ISSUE TITLE
mango groves don't show yield

### ISSUE THREAD

**Issue by MrZorG33** [We-the-People-civ4col-mod/Mod#867]
mango groves located on LR do not show yield.
![image](https://github.com/We-the-People-civ4col-mod/Mod/assets/61798883/b561473c-1c4c-4a21-b61b-7ac1c0a8ce5f)

and on the screen of the colony, the situation is even stranger. if you move a colonist to a tile with a mango grove, the extraction of the resource that he extracted on the previous tile will be shown.
![image](https://github.com/We-the-People-civ4col-mod/Mod/assets/61798883/67580b46-595b-4ea7-8a32-74d0659fc3d4)
![image](https://github.com/We-the-People-civ4col-mod/Mod/assets/61798883/c7633bce-0a6a-4876-9544-4f241b0565e1)

![image](https://github.com/We-the-People-civ4col-mod/Mod/assets/61798883/68939731-c48d-4f47-b202-197d0a8a08e6)
![image](https://github.com/We-the-People-civ4col-mod/Mod/assets/61798883/3ca57d1d-8561-4493-9408-103eac4d5958)

**Comment by raystuttgart** [comment]
**@Mr. Zorg:**
That was also already answered.

Problem is that there are no Professions that allow cutting Wood on Water.
(And doing / allowing so would most likely be an immersion issue.)

Mangroves **on Large Rivers** was not really supposed to have any Yields when I implemented / allowed it.
They were **intended mainly as an obstacle** on Large Rivers to be **removed by Pioneers** - for immersion.

**Simplest solution would simply be:**
**Remove mangroves** from Large Rivers in XML.
Because it seems to repeatedly confuse people and require discussion again and again.

**Summary:**
It is not really a bug ... but people expect stuff that I just never tried to implement.

**In other words:**
If something that I implemented as small immersion gimmick is perceived as a bug, remove it please.
(Because the only reason I allowed **Mangroves on LargeRivers** was **for a bit of flavour**.)

I understand that it just does not do what players expect.
But otherwise, let us please not call it a bug ...

Since I am not working on the mod currently, I will leave it to the team to do what it prefers.
(Alternatively it could also be configured in the Professions that cutting Woods on Water is possible ... )

**Comment by MrZorG33** [comment]
I played a little on the map created by the map generator and came to the conclusion that this is not a problem - if necessary, the Pioneers clear the LR.
